Formula/avr-gcc.rb

```diff
@@ -69,6 +81,8 @@ def install
     info.rmtree
     man7.rmtree
 
+    localBuild = build
```

Comment:
why the need of a new variables?

Reply:
This is needed, since, for some reason i don't know exactly, i don't have access to the build options inside of the resource-entity. But if i save the reference to the build options in a local variable i can access it. Maybe someone with more knowledge about ruby than me knows a more straight solution to this.

Reply:
we'll keep it that way then and I'll investigate in the future.

Reply:
Ah, wait, i didn't see your question in the context of the changed file. You're right, the second assignment to the localBuild-Variable is not needed. I will change that.

Reply:
Tested it, the second assignment is really needed. That puzzles me.